swarm/api/http/server.go

```diff
@@ -644,10 +649,15 @@ func (s *Server) logError(format string, v ...interface{}) {
 
 func (s *Server) BadRequest(w http.ResponseWriter, r *Request, reason string) {
 	s.logDebug("bad request %s %s: %s", r.Method, r.uri, reason)
```

Comment:
In general can we remove the logging from before the showError calls. 
and put a debug log in the showError